### ISSUE TITLE
Exception

### ISSUE THREAD

**Issue by coder1968** [yarrm80s/orpheusdl-applemusic-basic#3]
When I try to use this it says:

Exception: applemusic does not support track downloading

I used the formatting from the example with the -sd option specifying the module that should download.

**Comment by Dniel97** [comment]
```
An Apple Music module for the OrpheusDL modular archival music program, for playlists, lyrics and covers only
```

**Comment by coder1968** [comment]
Thanks for your reply.  I thought I understand its capabilities but perhaps my understanding is wrong.  I was trying to use Apple as a source for a playlist but download the actual songs from another source that supports the downloading of music files. I tried various ones and followed this reference in the readme

Just call orpheus.py with any playlist you want to archive, along with specifying a separate download module:
python orpheus.py https://music.apple.com/us/playlist/beat-saber-x-monstercat/pl.0ccb67a275dc416c9dadd6fe1f80d518 -sd qobuz8

The output was:

orpheusdl\orpheus\core.py", line 365, in orpheus_core_download
    raise Exception(f'{mainmodule} does not support track downloading') # TODO: replace with ModuleDoesNotSupportAbility
Exception: applemusic does not support track downloading

Thanks

**Comment by OrfiDev** [comment]
@coder1968 I completely forgot that I haven't fixed that bug, currently that capability won't work, sorry, it'll be fixed eventually... maybe

**Comment by retinoic** [comment]
> @coder1968 I completely forgot that I haven't fixed that bug, currently that capability won't work, sorry, it'll be fixed eventually... maybe

Does this module work at all then (just for grabbing playlist/lyrics/covers)? Attempting to call it without the -sd option also results in the same error.

**Comment by OrfiDev** [comment]
@retinoic Yes, however you need to use it indirectly, as the module set to download lyrics or covers.

**Comment by retinoic** [comment]
@OrfiDev ah, okay, thanks for clarifying - this makes more sense now.
